test/TestPrometheusMiddleware.AspNetCore.2.1/TestPrometheusMiddleware.AspNetCore.2.0.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'OpenCensus.sln'))\build\Common.test.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
testing of 2.1 is enough. 2.0 has reached EOL:

https://dotnet.microsoft.com/platform/support-policy/dotnet-core

![image](https://user-images.githubusercontent.com/9950081/51490337-d722ee00-1d5f-11e9-9912-8e78876a6194.png)